topology/sof-apl-da7219.m4

```diff
@@ -32,26 +32,26 @@ DEBUG_START
 # Schedule 48 frames per 1000us deadline on core 0 with priority 0
 PIPELINE_PCM_ADD(sof/pipe-volume-playback.m4,
 	1, 0, 2, s32le,
-	48, 1000, 0, 0)
+	48, 1000, 0, 0, 48000, 48000)
```

Comment:
@juimonen can we remove the 48 here, since it can be derived now from the min/max rates. This may need an ABI update and a pipeline update since we will be sending min/max rate with topology and runtime rate via hw_params.

Reply:
@lgirdwood sorry I didn't get it...? isn't the 48 like frames per scheduling slice (1000 us in this case). Min and max rates define the range of sample rates that the pipeline accepts. So, I mean aren't these values orthogonal? For e.g. I could have range 8000-192000Hz and scheduling of, say, 108 frames per 1020 us?

Reply:
@juimonen I think the logic is that we have a constant sample rate and a schedule time, meaning our processing size is always rate / schedule time.
Variable sample rate components would know their schedule time and would have to determine how many samples to produce/consume every time they are scheduled. Does this make sense ?

Reply:
@juimonen the intention is that the real run time rate will be passed in during hw_params(), this is then propagated through the pipeline from host component to DAI. The components will then have a chance to calculate samples per schedule based on the real rate. If an SRC is in the pipeline then it can alter the rate downstream or upstream of it so the subsequent components will use the SRC rate.